docs/1_Core/Technical_Report_Writing/03_Sample_Report/index.md

```diff
@@ -1,5 +1,8 @@
-# Sample Report
+# Sample Report and Slides
 
-[Sample Report Feedback from Turnitin](Report_Feedback_Compressed.pdf)
+- Report - 1 : [The Problems of the Web and How to fix them](Report_Feedback_Compressed.pdf)
+- Report - 2 : [CyberSecurity and Data Leak Detection](https://drive.google.com/file/d/1QrA23Df_pymzKW4TxfoeI4YXoxhEEnT_/view?usp=share_link)
 
-Observe the feedback from the professors, and improve on those points.
+- Slides - 2 : [CyberSecurity and Data Leak Detection - Slides](https://drive.google.com/file/d/1JDh-URd5JPajHgbjY9BUCSKOfaa5_-bx/view?usp=share_link)
+
+Observe the feedback from the professors, and improve on those points. Report - 2 has been provided by a junior, this report and presentation have been graded the highest for IInd Sem, 2022-23 . Thank you Sivaa! ✨
```

Comment:
My man put "thank you sivaa" on his own. Idk what more to say 😭😂

Reply:
Classic, I noticed I put my Github Repo with lab codes on it for CSF111 back when I started (where I did not mention myself). You had edited it to that line. So, I copy pasted lol.